### DISCUSSION TITLE
Uninstallation of FreeRDP built from (Git) source

### DISCUSSION THREAD

**Discussion by tukusejssirs** [FreeRDP/FreeRDP#7178] [Q&A] [answered]
Some years ago, we could simply execute `xargs rm < install_manifest.txt` (as suggested by @dorianj [here](https://github.com/FreeRDP/FreeRDP/issues/492#issuecomment-4532318)). But [this commit](https://github.com/FreeRDP/FreeRDP/commit/db86291c038acb22426aa5b4a69efac077d050bf) added `install_manifest.txt` to ignored files by Git.

Is there a (presumably) clean way to uninstall FreeRDP built from source?

**Comment by akallabeth** [comment, marked as answer]
@tukusejssirs what is the problem with `install_manifest.txt` being ignored by git, how does that influence your `uninstall`?

**Reply by tukusejssirs** [reply]
@akallabeth, I think that by your comment you suggest that I can use `sudo make uninstall` to uninstall FreeRDP built from source; is it true?

Just to answer your question: without `install_manifest.txt` (which contains list of files created by FreeRDP installation) I don’t know which files to delete in order to uninstall FreeRDP built from source.

**Reply by akallabeth** [reply]
@tukusejssirs no, you can´t just run `make uninstall` but `install_manifest.txt` is still there (created during compile). the commit you reference just adds the file to the `gitignore` list (so that it will not be checked in as it is generated on the fly)

**Reply by tukusejssirs** [reply]
Oh, sorry, I didn’t noticed that! Thanks! :+1:
